src/utils/backend.js

```diff
@@ -45,8 +46,8 @@ export default class Backend {
 
   static sendProfileData = async (address, postParam) => backendFetch(`profile/${address}`, {
     method: 'post',
-    body: JSON.stringify(postParam),
-    headers: { 'Content-Type': 'application/json' },
+    body: postParam instanceof FormData ? postParam : JSON.stringify(postParam),
+    ...!(postParam instanceof FormData) && { headers: { 'Content-Type': 'application/json' } },
```

Comment:
we can also extract condition 🤩
```suggestion
    ...postParam instanceof FormData
      ? { body: postParam }
      : {
        body: JSON.stringify(postParam),
        headers: { 'Content-Type': 'application/json' }
      },
```
@mradkov can you squash all commits if you decide to apply this one?

Reply:
I will apply it and squash.